sdk/src/main/scala/com/horizen/account/AccountSidechainNodeViewHolder.scala

```diff
@@ -206,6 +269,7 @@ class AccountSidechainNodeViewHolder(sidechainSettings: SidechainSettings,
       }
     }.foreach(tx => newMemPool.put(tx))
     newMemPool
+
```

Comment:
In SidechainNodeViewHolder a mailbox ("akka.actor.deployment.prio-mailbox") was defined for the actor. Should we add it also here?

Reply:
yes

Reply:
address in https://github.com/HorizenOfficial/Sidechains-SDK/pull/560